app/views/returns/DirectlyExportedComponentsView.scala.html

```diff
@@ -34,7 +34,7 @@
     @paragraphBody(messages("directlyExportedComponents.paragraph.2"))
 }
 
-@layout(pageTitle = title(form, messages("directlyExportedComponents.title", totalPlastic), Some(messages("return.title.tail")))) {
+@layout(pageTitle = title(form, messages("directlyExportedComponents.title", totalPlastic.asKgs), Some(messages("return.title.tail")))) {
```

Comment:
if kgs is incorrect coisider changing to asKg - however not a priority at all.

Reply:
done it anyway :)